roles/gpu_operator_bundle_from_commit/files/operator_bundle_image_builder_script.yml

```diff
@@ -15,6 +15,12 @@ data:
     echo "OPERATOR_IMAGE_NAME=${OPERATOR_IMAGE_NAME}"
 
     CONTAINER_FILE=./bundle.Dockerfile
+    if [ ! -f ${CONTAINER_FILE} ]; then
```

Comment:
@omertuc is this the change that caused the failures, or was this not picked up by the last run? I have to step away for the evening, but will look at this again first thing tomorrow.

Reply:
Nah it's just the CI being weird, retrying

Reply:
We'll [know](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/211/pull-ci-openshift-psap-ci-artifacts-master-images/1415032472961290240) in 5m